portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -30,7 +30,7 @@ public int getTextLength() {
       return (Integer) this.text.length();
   }
 
-  public String getFilter() {
+  /*public String getFilter() {
```

Comment:
Please remove outcommented code. You can always bring it back from revision history.